flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -737,7 +738,8 @@ class ExprLowering {
           [&](Fortran::lower::FirOpBuilder &builder) {
             auto str = consLit();
             builder.create<fir::HasValueOp>(getLoc(), str);
-          }, builder.createLinkOnceLinkage());
+          },
+          builder.createLinkOnceLinkage());
```

Comment:
clang-format is weird

Reply:
I was going to manually revert that change but I see you have the same change in your PR #315, so looks like a clang-format change or something like that.